invenio_pidrelations/contrib/versioning.py

```diff
@@ -176,17 +176,23 @@ def pid_version_parent(child):
     return PIDVersioning(child=child).parent
 
 
-@versioning_blueprint.app_template_test()
-def latest_version(child_pid=None, parent_pid=None):
-    """Determine if PID is the last version."""
-    assert child_pid or parent_pid
-    return PIDVersioning(child=child_pid, parent=parent_pid).last_child
+@versioning_blueprint.app_template_filter()
+def last_version(pid, is_sibling=False):
+    """Get Last version of a pid."""
+    if is_sibling:
+        return PIDVersioning(child=pid).last_child
+    else:
+        return PIDVersioning(parent=pid).last_child
 
 
 @versioning_blueprint.app_template_filter()
-def pid_versions(pid):
+def pid_versions(pid, is_sibling=False):
```

Comment:
The default of this parameter should perhaps be ``True``, as most of the times you would render it on a page of a sibling, not a parent. Also, this way it would be consistent with the previous signature.